### PR TITLE
Added create view for product

### PR DESCRIPTION
**Pull request by JuanSaieh**
Changes:
- Added create view for product with styes
Screen Shot 2019-08-02 at 4 45 34 PM
https://trello.com/c/ZaCNffDx

### CODE REVIEW

File: app/controllers/products_controller.rb [hunk 1]
```diff
@@ -4,4 +4,29 @@ def index
...
   end
 
+
+  def new
+    @product = Product.new
+  end
+
+  def create
+    @product = Product.new(product_params)
+
```
Comment (Jucerweb) [inline app/controllers/products_controller.rb]: Spaces

File: app/controllers/products_controller.rb [hunk 2]
```diff
@@ -4,4 +4,29 @@ def index
...
+      redirect_to products_path, notice: 'your User is now live'
+    else
+      render :new
+    end
+   
+    
+  end
+
+  def product_params
+    product_params = params.require(:product).permit(:name,
```
Comment (Jucerweb) [inline app/controllers/products_controller.rb]: Please indent this

File: app/controllers/products_controller.rb (hunk 1 above)
Comment (JuanSaieh) [reply app/controllers/products_controller.rb]: Done!

File: app/controllers/products_controller.rb (hunk 2 above)
Comment (JuanSaieh) [reply app/controllers/products_controller.rb]: done!

File: config/routes.rb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  resources :products, only: [:index, :destroy]
+  resources :products, only: [:index, :new, :create, :destroy]
```
Comment (Jucerweb) [inline config/routes.rb]: Use except

File: app/views/products/index.html.erb [hunk 4]
```diff
@@ -1 +1,22 @@
...
+      <th scope="col">Quantity</th>
+      <th scope="col">Price</th>
+      <th scope="col">Action</th>
+    </tr>
+  </thead>
+  <tbody>
+    <%= render partial: 'product', collection: @products %>
+  </tbody>
+</table>
+
```
Comment (Jucerweb) [inline app/views/products/index.html.erb]: Delete these spaces

File: app/views/products/_form.html.erb [hunk 5]
```diff
@@ -0,0 +1,29 @@
+<%= form_with(model: product, local: true, class: "col-md-6 offset-md-3 text-center" ) do |form| %>
+
+  <div class="form-group col-md-12">
+  <%= form.label :name %>
```
Comment (Jucerweb) [inline app/views/products/_form.html.erb]: Identation

File: config/routes.rb (hunk 3 above)
Comment (JuanSaieh) [reply config/routes.rb]: Done!

File: app/views/products/index.html.erb (hunk 4 above)
Comment (JuanSaieh) [reply app/views/products/index.html.erb]: Done!

File: app/views/products/_form.html.erb (hunk 5 above)
Comment (JuanSaieh) [reply app/views/products/_form.html.erb]: Done!

**Pull request merged**